examples/jpa-hibernate-arquillian/README.adoc

```diff
@@ -0,0 +1,323 @@
+            cq.where(condition);
+        }
+        return entityManager.createQuery(cq).getSingleResult().intValue();
+    }
+
+}
+----
+
+== persistence.xml
+
+Here we define which database will persist our movies, and we perform other configurations such as: define a persistence-unit with the name movie-unit, followed by the definition of the JPA provider/implementation (in this case Hibernate 5) and we set some properties for hibernate 5:
+
+----
+ <persistence xmlns="http://java.sun.com/xml/ns/persistence"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd"
+             version="2.0">
+    <persistence-unit name="movie-unit">
+        <provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
+        <jta-data-source>movieDatabase</jta-data-source>
+        <non-jta-data-source>movieDatabaseUnmanaged</non-jta-data-source>
+        <class>org.superbiz.injection.h3jpa.Movie</class>
+        <properties>
+            <property name="hibernate.hbm2ddl.auto" value="update"/>
+            <property name="hibernate.dialect" value="org.hibernate.dialect.HSQLDialect"/>
+            <property name="hibernate.show_sql" value="true"/>
+
+            <!--
+            JPA and CDI are linked, enabling JPA to use CDI for its
+            components but CDI can use JPA too. To solve issues with
+            hibernate you need to this property either as system property
+            or persistence unit
+            -->
+            <property name="tomee.jpa.factory.lazy" value="true"/>
+        </properties>
+    </persistence-unit>
+</persistence>
+----
+
+
+== arquillian.xml
+This file provides the configuration the server will have for running the tests.
+The property `additionalLibs` provide to the server the jar files required for Hibernate 5 as explained in the http://tomee.apache.org/latest/docs/tomee-and-hibernate.html[TomEE and Hibernate] documentation.
+
+
+----
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<arquillian xmlns="http://jboss.org/schema/arquillian"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+    <container qualifier="tomee" default="true">
+        <configuration>
+            <property name="httpPort">-1</property>
+            <property name="ajpPort">-1</property>
+            <property name="stopPort">-1</property>
+            <property name="dir">target/tomee-remote</property>
+            <property name="appWorkingDir">target/arquillian-remote-working-dir</property>
+            <property name="cleanOnStartUp">true</property>
+            <property name="additionalLibs">
+                <!-- add hibernate 5 need it jars to the server-->
```

Comment:
Question: Shouldn't we go for `5.4.31.Final` here as it contains several (query) related bug fixes since `5.4.10.Final` (just in case a user is going to simply CnP the example) ?

Reply:
The version I added in the example were taken from the test we currently have for the integration: 

https://github.com/apache/tomee/blob/master/arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/java/org/apache/openejb/arquillian/tests/cmp/sample/CustomOrmXmlHibernateEarTest.java#L66-L75

We can open a new PR after this one to test and update to the latest version in a wider scope that includes the arquillian-tomee-test module along with the rest of the examples and references to a newer version.